docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
+  goravel:
+    build:
+      context: .
+        ports:
+          -"3000:3000"
+        restart: always
```

Comment:
There is a problem with indentation and please create your branch based on master.